native/src/main/java/io/ballerina/lib/soap/Util.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2024 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.lib.soap;
+
+import io.ballerina.runtime.api.values.BError;
+
+/**
+ * Util methods for SOAP operations.
+ */
+public class Util {
+    private Util() {
+        //private constructor
+    }
+
+    public static void notifyFailure(BError bError) {
```

Comment:
Do we need separate class here. we can simple use bError.printStackTrace();

Reply:
Need to add  System.exit(1);

Reply:
I'll fix that.

Reply:
`System.exit(1)` gives a spotbugs error which is why I removed it.

Reply:
Fixed with 752f2efd1511410cd25dd5d6a3131794db216c53